<launch>
    <node name="serial_node"        pkg="rosserial_python"      type="serial_node.py" output="screen">
    <param name="port"              type="string"               value="/dev/ttyACM0"/>
    </node>

    <node pkg="trilobot" name="battery_bridge" type="battery_bridge.py" output="screen"/>


</launch>
